<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.apache.kafka.common.security.oauthbearer.internals.OAuthBearerSaslClientTest" tests="3" skipped="0" failures="0" errors="0" timestamp="2022-09-03T10:38:33" hostname="LAPTOP-FH01OFOM" time="0.006">
  <properties/>
  <testcase name="testWrapsExtensionsCallbackHandlingErrorInSaslExceptionInFirstClientMessage" classname="org.apache.kafka.common.security.oauthbearer.internals.OAuthBearerSaslClientTest" time="0.004"/>
  <testcase name="testNoExtensionsDoesNotAttachAnythingToFirstClientMessage" classname="org.apache.kafka.common.security.oauthbearer.internals.OAuthBearerSaslClientTest" time="0.002"/>
  <testcase name="testAttachesExtensionsToFirstClientMessage" classname="org.apache.kafka.common.security.oauthbearer.internals.OAuthBearerSaslClientTest" time="0.0"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
